version1/02 - Object types.md

```diff
@@ -90,863 +94,639 @@ The following notation is used in the tables below:
-|           | ref       | refRegistreringSomSekundaer   |          |             |                   |  |      |      |   x   |      |  x   |   x    | <p>&bull; references **AbstraktRegistrering** as a secondary **Klasse**</p><p>&bull; **refBarnKlasse** cannot be set if **refRegistreringSomSekundaer** is set</p> |
+|           | ref       | refKlassifikasjonssystem      |    V     |             |                   |  |      |  V   |   V   |      |  V   |   V    |                                                                                                                                                                    |
+|           | ref       | refForelderKlasse             |          |             |                   |  |      |  V   |   V   |      |  V   |   V    |                                                                                                                                                                    |
+|           | ref       | refBarnKlasse                 |          |             |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; Cannot have both refBarnKlasse and kassasjonsvedtak</p>                                                                                                  |
+|           | ref       | refMappeSomPrimaer            |          |             |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **AbstraktMappe** as a primary **Klasse**</p> <p>&bull; **refBarnKlasse** cannot be set if **refMappeSomPrimaer** is set</p>                  |
+|           | ref       | refMappeSomSekundaer          |          |             |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **AbstraktMappe** as a secondary **Klasse**</p> <p>&bull; **refBarnKlasse** cannot be set if **refMappeSomSekundaer** is set</p>              |
+|           | ref       | refRegistreringSomPrimaer     |          |             |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **AbstraktRegistrering** as a primary **Klasse**</p> <p>&bull; **refBarnKlasse** cannot be set if **refRegistreringSomPrimaer** is set</p>    |
+|           | ref       | refRegistreringSomSekundaer   |          |             |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **AbstraktRegistrering** as a secondary **Klasse**</p><p>&bull; **refBarnKlasse** cannot be set if **refRegistreringSomSekundaer** is set</p> |
 
 ## Folders
 
-### Mappe
-
-| Field no. | Type      | Field                            | Not null | Has default | Code list         |  | Save | View | Query | Sort | Link | Unlink | Comment                                                                                                                         |
+### File ([Finalizable Object](#finalizable-object-basic-archival-object))
```

Comment:
I think this should become `Abstract mappe` and you should delete the `Abstract mappe` section which is a bit below.